opennms-dao/src/main/java/org/opennms/netmgt/dao/hibernate/AbstractDaoHibernate.java

```diff
@@ -30,11 +30,11 @@
 
 import java.io.Serializable;
 import java.sql.SQLException;
-import java.util.Collection;
-import java.util.List;
+import java.util.*;
```

Comment:
Don't use wildcard imports, should be updated with intellij Settings.

Reply:
changed intellije settings and wildcard imports removed